app/models/user.py

```diff
@@ -23,9 +25,15 @@ class Role(db.Model):
     @staticmethod
     def insert_roles():
         roles = {
-            'User': (
+            'Inactive': (
```

Comment:
Nothing. It's the old 'User' role. The only reason it's still there was because I couldn't figure out what role to make 'Default'